ironic-source-list

```diff
@@ -1,12 +1,20 @@
 crudini
 gunicorn
 {% if env.IRONIC_SOURCE %}
+    {% if path.isdir('/sources/' + env.IRONIC_SOURCE) %}
+git+file:///sources/{{ env.IRONIC_SOURCE }}
+    {% else %}
 ironic @ git+https://opendev.org/openstack/ironic@{{ env.IRONIC_SOURCE }}
+    {% endif %}
 {% else %}
 ironic @ git+https://opendev.org/openstack/ironic
 {% endif %}
 {% if env.IRONIC_INSPECTOR_SOURCE %}
+    {% if if path.isdir('/sources/' + env.IRONIC_INSPECTOR_SOURCE) %}
```

Comment:
Am I seeing double `if` and that is on purpose?

Reply:
oh snap, thanks for spotting this!

Reply:
this is disturbing though, I tested the change and it works even with double if :/
anyway, I'll fix it

Reply:
strange, but anyways IMO it is safer if we fix it right away. Thanks